os_migrate/plugins/module_utils/server_port.py

```diff
@@ -62,7 +62,9 @@ class ServerPort(resource.Resource):
         'fixed_ips_refs',
         'network_ref',
     ]
-    sdk_params_from_params = []
+    params_from_sdk = [
+        'binding_profile'
```

Comment:
In a follow up PR we should add mac address here, that was one of the use cases i think.